packages/aws-cdk/lib/cdk-toolkit.ts

```diff
@@ -874,6 +874,63 @@ export class CdkToolkit {
       stackName: assetNode.parentStack.stackName,
     }));
   }
+
+  /**
+   * Checks to see if a migrate.json file exists. If it does and the source is either `filepath` or
+   * is in the same environment as the stack deployment, a new stack is created and the resources are
+   * migrated to the stack using an IMPORT changeset. The normal deployment will resume after this is complete
+   * to add back in any outputs and the CDKMetadata.
+   */
+  private async tryMigrateResources(stacks: StackCollection, options: DeployOptions): Promise<void> {
+    const stack = stacks.stackArtifacts[0];
```

Comment:
what about the other stacks?

Reply:
CDK migrate only creates one stack. It is not compatible with multiple stacks at this time.